backend/tests/v5/unit/services/sso/sso.constants.test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { src } = require('../../../helper/path');
+const { generateRandomString, determineTestGroup } = require('../../../helper/services');
+
+const SSOConstants = require(`${src}/services/sso/sso.constants`);
+
+const testGetProviderLabel = () => {
+	describe('Get Provider label', () => {
+		test('All known providers should return a label', () => {
+			Object.values(SSOConstants.providers).forEach((provider) => {
+				expect(SSOConstants.getProviderLabel(provider)).not.toBeUndefined();
```

Comment:
I don't think there's much point of this.

that suggest we'd hard code the values here to make sure it match.

The alternative would be to expose the value and match it.

If someone changed the label, it would've gone through the code review. We just need to make sure there's always a label for each SSO type